projects/packages/forms/src/contact-form/class-admin.php

```diff
@@ -1316,6 +1316,29 @@ public function grunion_add_admin_scripts() {
 			)
 		);
 
+		// Add tracking scripts
+		wp_enqueue_script( 'jp-tracks', '//stats.wp.com/w.js', array(), gmdate( 'YW' ), true );
```

Comment:
Could we rely on `Tracking::register_tracks_functions_scripts()` instead?

Reply:
Thanks @jeherve! Yes, I was considering shifting to that approach today when diving back in, and will definitely do so now :).

For my own future reference and others, see tracking [docs](https://github.com/Automattic/jetpack/blob/c34478bb12f44b283dee1e9b879c42687e18bf02/projects/packages/connection/TRACKING.md).